### DISCUSSION TITLE
Making violin plots

### DISCUSSION THREAD

**Discussion by snwnde** [Samreay/ChainConsumer#117] [Ideas]
My flawed memory told me I could make violin plots with `ChainConsumer`. It seems not in reality.

Does anyone share the need of making violin plots with `ChainConsumer`? If so I'd be glad to draft its implementation.

(What's the gain compared to use https://seaborn.pydata.org/generated/seaborn.violinplot.html?)

**Comment by Samreay** [comment]
The closest is the get distributions function, which I think normally is a more useful plot as it allow comparison of the PDFs directly a bit nicer. The main concern with using something like seaborns violin plots in my mind are: 1) We probably dont want to use their kernel smoothing because it normally over smooths and 2) It adds another dependency to chainconsumer, and the lighter chainconsumer is, the easier it is to install without it causing dependency versioning issues

**Reply by snwnde** [reply]
Thanks! Agreed about the concern on using seaborns. I will have to experience and think more if I could just use 1d distributions rather than violin plots.
